src/Sfa.Tl.Matching.Web/Views/ProviderVenue/ConfirmProviderVenueChange.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@model Sfa.Tl.Matching.Models.ViewModel.HideProviderVenueViewModel
 
 @{
-    ViewData["Title"] = (Model.IsEnabledForSearch ? "Hide" : "Unhide") + Model.Postcode;
+    ViewData["Title"] = (Model.IsEnabledForSearch ? "Hide " : "Unhide ") + Model.Postcode;
```

Comment:
Title doesn't Match H1

Reply:
The Title and  H1 are different according to the copy deck